### ISSUE TITLE
First pass train model

### ISSUE THREAD

**Issue by aamster** [AllenInstitute/DeepCell#33]
Tune hyperparameters such that model achieves maximum mean F1 across CV folds

Report:
- training loss curve
- train/val F1 curve
- confusion matrix
- mean val precision/recall across all ROIs, val precision/recall per experiment, val precision/recall per experiment by depth
- Mistake trends. Identify cases where the model disagrees with the label and try to identify any trends in these mistakes. 

**Tasks**
- [ ] Train classifier with default hyperparameters
- [ ] Create grid of jobs exploring hyperparameter space around defaults. Re-train and select the best one.
- [ ] Report performance metrics as described above

**Acceptance criterion**
- [ ] We have a trained model configured with the hyperparameters selected as above and understand performance and limitations of the model

Once this ticket an #26 are completed, we should collate the results into a presentation to the stakeholders.
